specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json

```diff
@@ -1145,6 +1145,263 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceBus/namespaces/{namespaceName}/migrationConfigurations": {
+      "get": {
+        "tags": [
+          "migrationConfigurations"
+        ],
+        "operationId": "migrationConfigurations_List",
```

Comment:
>migrationConfigurations [](start = 24, length = 23)

please think about consistency in the operation naming. Here you have `migrationConfigurations` while the other already existing operation group is named `DisasterRecoveryConfigs`

Reply:
could you please help me on this? 
for DisasterRecoveryConfigs operations its 'DisasterRecoveryConfigs' for Namespaces operations its 'Namespaces'.

Reply:
the thing is that you have already shipped `DisasterRecoveryConfigs` (I mean shortened form of Configuration) that is why I advice keeping the consistency in names to shorten `migrationConfigurations` to `MigrationConfigs`

Reply:
@hovsepm change the shortened to 'MigrationConfigs'. Also in URI, DR is 'disasterrecoveryconfigs' so have changed for migration to 'migrationconfigs'. please let me know the URI change is fine or  need to revert it.